renderer/tdf/common/CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ GlobalPackages_Add(footstone)
 target_link_libraries(${PROJECT_NAME} PUBLIC footstone)
 # endregion
 
+# region vfs
```

Comment:
There is no `vfs` related part in the current commit, is there a necessary dep here?

Reply:
This later commit([91498c4](https://github.com/Tencent/Hippy/pull/2552/commits/91498c4d2b9c2e03130300d1117489282237add9)) has used the vfs' UriLoader